### PR TITLE
Upadte from upstream

### PR DESCRIPTION
**Pull request by e-kov**
There are improvements for Fedora build, fix for kernel 5.4 etc.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -132,6 +132,9 @@ steps:
 
 - name: trigger
   image: plugins/downstream
+  # This step fails when 'packaging' hasn't a branch with the
```
Comment (anelson) [inline .drone.yml]: Why is this ignored?  Packaging will never have a branch with the same name except on `master`.

File: BUILDING-PACKAGE.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # Building the package
 
-This package is built using [debbuild](https://github.com/ascherer/debbuild) for Debian targets.
+This package is built using [debbuild](https://github.com/debbuild/debbuild) for Debian targets.
 
 The Makefile included in this project helps you to do builds, but you need
 to have debbuild installed first.
 
-You can install debbuild from the deb published on [the GitHub project releases](https://github.com/ascherer/debbuild/releases).
+You can install debbuild from the deb published on [the GitHub project releases](https://github.com/debbuild/debbuild/releases).
```
Comment (anelson) [inline BUILDING-PACKAGE.md]: Have we actually updated our build environment to use this new repo for `debbuild`?

File: .drone.yml (hunk 1 above)
Comment (e-kov) [reply .drone.yml]: For instance, I'm not going to have a packages repository for some assurio-snap branch. And I'll not create appropriate packaging branch. Why build have to be failed in this case?
If we need some "feature build" packages, then I'll just branch master in the packaging with the same branch name.

File: BUILDING-PACKAGE.md (hunk 2 above)
Comment (e-kov) [reply BUILDING-PACKAGE.md]: Actually, ascherer is the main contributor to the debbuild. So both links are good. But anyway, good point. PR has been [updated](https://github.com/assuriosw/assurio-snap/pull/10/commits/137a08add13aba76c7d8eab26f2e53643f96ffba).
And packaging [too](https://github.com/assuriosw/packaging/pull/4)

File: .drone.yml (hunk 1 above)
Comment (anelson) [reply .drone.yml]: Oh.  I didn't realize that's how your packaging scripts work.  That's not acceptable.  Please create a git issue in the `packaging` repo about the problem that there must be a matching branch in `packaging` for every branch build in every other repo.

Once you do that I'll approve this.

Comment (e-kov) [reply .drone.yml]: https://github.com/assuriosw/packaging/issues/5

**Pull request merged**